CONTRIBUTING

```diff
@@ -19,8 +19,14 @@ guidelines:
     1. Anything other than a trivial contribution requires a Contributor
     License Agreement (CLA), giving us permission to use your code. See
     https://www.openssl.org/policies/cla.html for details.  If your
-    contribution is too small to require a CLA, put "CLA: trivial" on a
-    line by itself in your commit message body.
+    contribution is too small to require a CLA (e.g. fixing a spelling
+    mistake), put "CLA: trivial" on a line by itself in your commit message
+    body (not in the pull request description).  To add the "CLA: trivial"
+    line after submission, do the following:
```

Comment:
Often it is necessary to mention the empty line which goes between commit message _title_ and _body_. Maybe something like

```
contribution is too small to require a CLA (e.g. fixing a spelling
mistake), place the text "CLA: trivial" on a line by itself in the
body of your commit message (separated from the commit title by an
empty line). It is not sufficient to place the text in the GitHub
pull request description only.

To amend a missing "CLA: trivial" line after submission, do the following:
```

Reply:
I would say "separated with an empty line from the rest of the commit message", so no one starts believing it must come directly efter the subject line.

Reply:
I've added bits of both suggestions and fiddled with things a bit.